src/app-layer-parser.c

```diff
@@ -655,7 +662,7 @@ uint64_t AppLayerTransactionGetActiveLogOnly(Flow *f, uint8_t flags)
     AppLayerParserProtoCtx *p = &alp_ctx.ctxs[f->protomap][f->alproto];
 
     if (p->logger == TRUE) {
```

Comment:
On 17-12-15 16:36, Jason Ish wrote:
>> @@ -655,7 +662,7 @@ uint64_t AppLayerTransactionGetActiveLogOnly(Flow *f, uint8_t flags)
>>      AppLayerParserProtoCtx *p = &alp_ctx.ctxs[f->protomap][f->alproto];
>>  
>>      if (p->logger == TRUE) {
> 
> The logger is for both directions, you can't control that. If you want
> it for one direction you have to ignore a certain direction in your log
> callback. So nothing has really changed here. On transaction complete,
> both log IDs will be incremented and we'll move forward.

The problem is that both log_id's will now be used. Are both updated in
all possible configurations?